DemonsDiseases/Dabbas.xml

```diff
@@ -0,0 +1,51 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="Dabbas" xml:lang="en" type="auth">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>dabbās</title>
```

Comment:
If we want to capitalise demons as they are persons, we should capitalise Dabbās, as we did with Budā (even if this is not necessarily a proper demon) and Bāryā.